Design.md

```diff
@@ -238,6 +258,12 @@ This API query S3 credentials of the  user using a `filter` parameter.
 ### List S3 Credentials
 This API list S3 credentials of the  user.
 1. `list-access-keys` api will be called using assumed role credentials.
+2. Retrieve the secret keys, formatted as `<Username>__<AccessKeyID>`, from Redis Sentinel.
```

Comment:
Do we send keys that are in `INACTIVE` state as well?

Reply:
Yes. This OSIS `listS3Credentials` API will return IAM `INACTIVE` the keys with the `active` flag as `false`